src/ducks/balance/components/HealthReimbursementsIcon.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Icon from 'cozy-ui/react/Icon'
+import cx from 'classnames'
+import healthCatIcon from 'assets/icons/categories/icon-cat-health.svg'
+import styles from 'ducks/balance/components/ReimbursementsIcon.styl'
+import { getCssVariableValue } from 'cozy-ui/react/utils/color'
+
+export default function ReimbursementsIcon(props) {
+  const { className, icon, ...rest } = props
+
+  return (
+    <span className={cx(styles.ReimbursementsIcon, className)}>
+      <Icon icon={icon} {...rest} />
+      <span className={cx(styles.ReimbursementsIcon__hourglass)}>
+        <Icon
+          icon="hourglass"
+          size={8}
+          color={getCssVariableValue('coolGrey')}
+        />
+      </span>
+    </span>
```

Comment:
I think the Badge component from cozy-ui can be helpful here.https://docs.cozy.io/cozy-ui/react/#badge

Reply:
We don't have the good style in Badge component. But I keep that in mind, maybe this component should be more flexible than it is.